lib/dojos/decline-join-request.js

```diff
@@ -0,0 +1,62 @@
+var _ = require('lodash');
+var async = require('async');
+function cmd_accept_user_request (args, done) {
+  var seneca = this;
+  var plugin = args.role;
+  var tokenData = args.data;
+  var currentUserId = args.user.id;
+  if (!args.user) return done();
+  var inviteTokenId = tokenData.inviteToken;
+  var requestedByUser = tokenData.requestedByUser;
+
+  async.waterfall([
+    loadUser,
+    verifyRequest,
+    removeJoinRequests
+  ], function (err, res) {
+    if (err) return done(null, {error: err.message});
+    return done(null, res);
+  });
+
+  function loadUser (done) {
```

Comment:
`verifyJoinRequest` perhaps? Sure, it loads a user, but that's just consequential. It's discarded afterwards.